README.md

```diff
@@ -53,7 +53,7 @@ care of setting up a production ready Highly Available cluster!
 ### Native Support For The Most Popular Providers
 
 KubeOne natively supports the most popular providers, including AWS, Azure,
-DigitalOcean, GCP, Hetzner Cloud, Nutanix, OpenStack, and VMware vSphere. 
+DigitalOcean, GCP, Hetzner Cloud, Nutanix, OpenStack, VMware Cloud Director, and VMware vSphere.
```

Comment:
Please also make a PR to update the docs.

Reply:
Yes I'll take care of that once all of this stuff is merged :)